.travis.yml

```diff
@@ -6,6 +6,9 @@ addons:
   firefox: latest
 before_install:
   - npm install -g greenkeeper-lockfile@1
-before_script: greenkeeper-lockfile-update
+before_script:
+  - greenkeeper-lockfile-update
+  - export CHROME_BIN=/usr/bin/google-chrome
```

Comment:
was it actually failing because these entries were not in the config? or is it the headless browsers?

Reply:
The Headless Chrome test were hanging on my local test run so I assume that there's something that changed with a Chrome update and one of our tests isn't completing.

Reply:
ahh I see